City-of-Pori/pori-kada
# PORI-225 fixed footer active trail styles, expanded visitpori footer …

…menus

To test:

1. drush fra -y && drush cc all
2. Make sure visitpori footer menus are expanded and the active and active trail icons are underlined with black border and not colored black. Also on mobile make sure the plus/minus icons look good with correct colors (white on red background and black on gray background)
